website/docs/components/link/inline/partials/guidelines/guidelines.md

```diff
@@ -14,7 +14,7 @@
 
 The Inline Link is only published as a component in Ember, there is no corresponding component in Figma. To achieve the same results, apply **text** and **color** styles from the [Foundations library](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=2130%3A2&t=6XBReWOxMRTiW1Iv-1) to your text.
 
-The Ember component inherits the styles of the text around it. Depending on the surrounding text, the link style in Figma may need to be overridden to ensure that the link and body text are the same.
+The Ember component inherits the typographic styles of the text around it (font, weight, etc). Depending on the surrounding text, the link style in Figma may need to be overridden to ensure that the link and body text are the same.
```

Comment:
This is helpful but since the consumer confusion this was in response to was around color it may be good to also specifically mention that it does _not_ inherit color.

Reply:
I had considered that and initially left it out since the section on color is directly adjacent. But, I guess since that was the case before and it was still missed or unclear it probably makes sense to include it. I've reworked the sentence a bit to communicate this, LMK what you think!

Reply:
Looks good, thanks for updating it!